components/RichText/RichText.tsx

```diff
@@ -110,16 +107,18 @@ export const RichText = ({
           anchor: linktype === 'story' ? anchor : '',
           // The WYSIWYG inline links automatically add a target="_self" by default which is unnecessary
           target: target === '_blank' ? '_blank' : undefined,
+          // Adding rel="noopener" for all eternal links that opens in new window/tab for security reasons
+          rel: linktype === 'url' && target === '_blank' ? 'noopener' : undefined,
         } as SbLinkType;
 
         return (
-          <SbLink
-            link={sbLink}
-            classes="group"
-            attributes={custom} // Custom link attributes
+          <CtaLink
+            sbLink={sbLink}
+            variant={linkColor === 'white' ? 'inline-white' : 'inline'}
+            className="*:inline"
           >
             {children}
-          </SbLink>
+          </CtaLink>
```

Comment:
Maps WYSIWYG links to CtaLink component variants - I created variants for inline light/dark mode links for use on dark basic card background colors